install/kubernetes/helm/istio/charts/sidecar-injector/templates/deployment.yaml

```diff
@@ -37,6 +39,24 @@ spec:
           - name: inject-config
             mountPath: /etc/istio/inject
             readOnly: true
+          livenessProbe:
+            exec:
+              command:
+                - /usr/local/bin/sidecar-injector
+                - probe
+                - --probe-path=/health
+                - --interval=2s
```

Comment:
Hmm... I have a general suggestion that we should rename this argument to "max_mtime_difference" or something specific instead of "interval", to avoid confusion.

Reply:
cc @jmuk